examples/ibm-transit-gateway/main.tf

```diff
@@ -36,3 +36,11 @@ resource "ibm_tg_connection" "test_ibm_tg_connection"{
 		name= "myconnection"
 		network_id = ibm_is_vpc.test_tg_vpc.resource_crn
 }
+// cross account
+resource "ibm_tg_connection" "test_ibm_tg_connection"{
+		gateway = "${ibm_tg_gateway.new_tg_gw.id}"
+		network_type = "vpc"
+		name= "mycrossconnection"
+		network_id = ibm_is_vpc.test_tg_vpc.resource_crn
```

Comment:
network_id this network id also should be other user account VPC?

ibm_is_vpc this will create the VPC in the same account